.eslintrc.json

```diff
@@ -15,7 +15,9 @@
             "files": ["**/*.test.ts?(x)"],
             "rules": {
                 // Ref.: https://github.com/typescript-eslint/typescript-eslint/issues/1776#issuecomment-601758481
-                "@typescript-eslint/unbound-method": "off"
+                "@typescript-eslint/unbound-method": "off",
+                // Allow expect.any(), expect.objectContaining(), expect.arrayContaining() usage in test files
+                "@typescript-eslint/no-unsafe-assignment": "off"
```

Comment:
I did not find a better way to get rid of eslint warning against the use of `expect.objectContaining` and `expect.arrayContaining`.

Another thing that eslint is warning against is the use of `expect.any(String)` in test files. Which is a great thing to do when we need it. That triggers a different eslint rule that I'd also consider overriding for test files, if we do not find a better way to make eslint compatible with it.